### PR TITLE
Avoid instantiating SwiftASTContextForModule in get_executable_triple…

### PR DESCRIPTION
**Pull request by adrian-prantl**
…() (NFC)

Instead we can directly query the triple of the module.

rdar://113997661

### CODE REVIEW

**Comment by adrian-prantl** [comment]
@swift-ci test

**Review by kastiglione — APPROVED** [review]

**Comment by adrian-prantl** [comment]
Added a bugfix for path remapping that was uncovered by this change!

**Comment by adrian-prantl** [comment]
@swift-ci test

**Comment by adrian-prantl** [comment]
@swift-ci test windows

**Comment by adrian-prantl** [comment]
@swift-ci test windows

File: lldb/test/API/lang/swift/clangimporter/Werror/TestSwiftStripWerror.py [hunk 1]
```diff
@@ -26,17 +26,13 @@ def test(self):
 
         # Turn on logging.
         log = self.getBuildArtifact("types.log")
-        self.expect("log enable lldb types -f " + log)
+        self.expect('log enable lldb types -f "%s"' % log)
```
Comment (kastiglione) [inline lldb/test/API/lang/swift/clangimporter/Werror/TestSwiftStripWerror.py:29]: we can use python f-strings now:
```suggestion
        self.expect(f'log enable lldb types -f "{log}"')
```

Comment (adrian-prantl) [reply lldb/test/API/lang/swift/clangimporter/Werror/TestSwiftStripWerror.py:29]: Nice, I did not know about this feature!

**Pull request merged**